common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1325,6 +1325,12 @@
     ],
     "sqlState" : "54001"
   },
+  "FEATURE_NOT_ENABLED" : {
+    "message" : [
+      "The feature <featureName> is not enable. Consider setting the config: <configKey> to <configValue>. To enable this capability."
```

Comment:
Typo: "not [enabled]"

Typo: "the config `<configKey>` to `<configValue>` to enable this" (remove `:` and `.`)